TransactionMobile/TransactionMobile/App.xaml.cs

```diff
@@ -34,6 +35,16 @@ public partial class App : Application
         /// </summary>
         public static TokenResponse TokenResponse;
 
+        /// <summary>
+        /// The estate identifier
+        /// </summary>
+        public static Guid EstateId;
+
+        /// <summary>
+        /// The merchant identifier
+        /// </summary>
+        public static Guid MerchantId;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5700460)